.github/workflows/cloud-workflow.yml

```
@@ -42,7 +42,7 @@ jobs:
             python3 build.py --generate
             sudo chown -R $USER $MAGMA_ROOT/*
             git add .
-            git status
+            echo GIT_STATUS=`git status` >> $GITHUB_ENV
```

Can you both save it and print it here (that way we can view it both from the Slack notification and in the job

---

Also, nit for uniformity:
```suggestion
            echo GIT_STATUS=$(git status) >> $GITHUB_ENV
```